components/Login/Login.js

```diff
@@ -0,0 +1,25 @@
+import { useSession, signIn, signOut } from "next-auth/react";
+import Image from "next/image";
+import { Button } from "./Login.styled";
+
+export default function Login() {
+  const { data: session } = useSession();
+  if (session) {
+    return (
+      <>
+        <Button onClick={() => signOut()}>
+          <Image src="/logout.svg" width={32} height={32} alt="logout" />
+          Sign out
+        </Button>
+      </>
+    );
+  }
```

Comment:
you could add an `else` here :)

Reply:
Thank you for your feedback i will add it now